pipelines/hacbs/hacbs-test.yaml

```diff
@@ -37,6 +37,22 @@
     workspaces:
     - name: workspace
       workspace: workspace
+- op: add
+  path: /spec/tasks/-
+  value:
+    name: deprecated-base-image-check
+    runAfter:
+      - sanity-inspect-image
+    taskRef:
+      name: deprecated-image-check
+    params:
+    - name: IMAGE_REGISTRY
+      value: registry.access.redhat.com
+    - name: IMAGE_REPOSITORY
+      value: $(params.git-url)
```

Comment:
Is this the source code url? But I think `IMAGE_REPOSITORY` should point to image repo, is it?

Reply:
@staticf0x yes, it should be `BASE_IMAGE_REPOSITORY` from the sanity-inspect-image task, so something like 
```
$(tasks.sanity-inspect-image.results.BASE_IMAGE_REPOSITORY)
```